ElectronClient/gui/KeymapConfig/KeymapConfigScreen.tsx

```diff
@@ -0,0 +1,144 @@
+	const styles = styles_(themeId);
+
+	const [filter, setFilter] = useState('');
+	const [keymap, setAccelerator, error] = useKeymap();
+	const [editing, enableEditing, disableEditing] = useEditing();
+
+	const handleSave = (event: { commandName: string, accelerator: string }) => {
+		const { commandName, accelerator } = event;
+		setAccelerator(commandName, accelerator);
+		disableEditing(commandName);
+	};
+
+	const hanldeReset = (event: { commandName: string }) => {
+		const { commandName } = event;
+		setAccelerator(commandName, keymapService.getDefaultAccelerator(commandName));
+		disableEditing(commandName);
+	};
+
+	const handleCancel = (event: { commandName: string }) => {
+		const { commandName } = event;
+		disableEditing(commandName);
+	};
+
+	const renderAccelerator = (accelerator: string) => {
+		return (
+			<div>
+				{accelerator.split('+').map(part => <kbd style={styles.kbd} key={part}>{part}</kbd>).reduce(
+					(accumulator, part) => (accumulator.length ? [...accumulator, ' + ', part] : [part]),
+					[]
+				)}
+			</div>
+		);
+	};
+
+	const renderStatus = (command: string) => {
+		if (!error) return null;
+		const { invalidAccelerator, invalidKeymapItem, duplicateAccelerators, duplicateKeymapItems } = error.details;
+
+		if ((invalidAccelerator && invalidKeymapItem.command === command) ||
+			(duplicateAccelerators && (duplicateKeymapItems[0].command === command || duplicateKeymapItems[1].command === command))) {
+			return '❌';
+		} else {
+			return null;
+		}
+	};
+
+	const renderKeymapRow = ({ command, accelerator }: KeymapItem) => {
+		const handleClick = () => enableEditing(command);
+		const cellContent = editing[command]
+			? <ShortcutRecorder
+				onSave={handleSave}
+				onReset={hanldeReset}
+				onCancel={handleCancel}
+				initialAccelerator={accelerator}
+				commandName={command}
+				themeId={themeId}
+			/>
+			: <div onClick={handleClick} style={styles.tableCell}>
+				<div style={styles.tableCellShortcut}>
+					{accelerator
+						? renderAccelerator(accelerator)
+						: <div style={styles.disabled}>{_('Disabled')}</div>
+					}
+				</div>
+				<div style={styles.tableCellStatus}>
+					{renderStatus(command)}
+				</div>
+			</div>;
+
+		return (
+			<tr key={command}>
+				<td style={styles.tableCommandColumn}>
+					{getLabel(command)}
+				</td>
+				<td style={styles.tableShortcutColumn}>
+					{cellContent}
+				</td>
+			</tr>
+		);
+	};
+
+	return (
+		<div>
+			{error &&
+				<div style={styles.warning}>
+					<p style={styles.text}>
+						<span>
+							{error.details.duplicateAccelerators
+								? _('Keymap configuration contains duplicates. Change or disable one of the shortcuts to continue.')
+								: error.details.invalidAccelerator
+									? _('Keymap configuration contains an invalid shortcut. Change or disable it to continue.')
+									: error.message // Highly unlikely that any other error will occur at this point
+							}
```

Comment:
These error messages should be part of the error object (in `error.message`), then here you can print error.message directly.